tests/issues/issue-250/formatted.pug

```diff
@@ -0,0 +1,7 @@
+div
+  v-progress-circular(
+    :color="getProgressColor(currentCapacity)",
+    :size="circleSize",
```

Comment:
I need to look into this later cause your assumption could be wrong
`:size=circleSize` should be converted to `:size=circleSize` and not `:size="circleSize"`, cause `circleSize` could be a variable in the pug file itself 🤔

Reply:
You are so right... that's what I get for reviewing code on a laptop while on vacation 😳.  Yes, what I did here was the wrong fix, they shouldn't have quotes, they should be left alone (still a corruption problem, but results need to be different).  I scanned up further in our source file and found this:
```pug
<template lang="pug">
  - const circleSize = 72;
  - const circleWidth = 7;
```
which is very atypical in our Vue code, but still makes sense here.